### ISSUE TITLE
Cannot convert PyTorch to ONNX using Pytorch 1.1 and ONNX 1.5.0

### ISSUE THREAD

**Issue by kalyc** [pytorch/pytorch#20031]
## 🐛 Bug

Cannot convert PyTorch to ONNX using the latest PyTorch and ONNX version

System Config:
```
pytorch                   1.1.0           py2.7_cuda10.0.130_cudnn7.5.1_0    pytorch
torchvision               0.2.2                      py_2    pytorch
cudatoolkit               10.0.130                      0  
```

## To Reproduce

1. Use Ubuntu 16.04
2. Install latest PyTorch conda package as mentioned on official website
3. pip install onnx==1.5.0

Run the following:

pytorch_export.py:
```
import os

import torch
import torch.nn as nn
import torch.nn.functional as F
import torch.optim as optim
from torchvision import datasets, transforms
from torch.autograd import Variable
import torch.onnx as torch_onnx

class Model(nn.Module):
    def __init__(self):
        super(Model, self).__init__()
        self.conv = nn.Conv2d(in_channels=3, out_channels=32, kernel_size=(3,3), stride=1, padding=0, bias=False)

    def forward(self, inputs):
        x = self.conv(inputs)
        return torch.mean(x, dim=2)

input_shape = (3, 100, 100)

curr_dir = os.path.dirname(__file__)
model_onnx_path = curr_dir + "/pytorch_model.onnx"

model = Model()
model.train(False)

dummy_input = Variable(torch.randn(1, *input_shape))
torch_onnx.export(model, 
                  dummy_input, 
                  model_onnx_path, 
                  verbose=True)

```

verify_onnx_model.py
```
import os

import onnx

curr_dir = os.path.dirname(__file__)
model_proto = onnx.load(curr_dir + "/pytorch_model.onnx")
onnx.checker.check_model(model_proto)

graph = model_proto.graph
inputs = []
for i in graph.input:
    inputs.append(i.name)
assert inputs == ['0', '1']

params = []
for tensor_vals in graph.initializer:
    params.append(tensor_vals.name)
assert params == ['1']

nodes = []
for node in graph.node:
    nodes.append(node.op_type)
assert nodes == ['Conv', 'ReduceMean']
```

```python pytorch_export.py```
```python verify_onnx_model.py```

Following error will be thrown:

```
++ python pytorch_export.py
graph(%0 : Float(1, 3, 100, 100),
      %conv.weight : Float(32, 3, 3, 3)):
  %2 : Float(1, 32, 98, 98) = onnx::Conv[dilations=[1, 1], group=1, kernel_shape=[3, 3], pads=[0, 0, 0, 0], strides=[1, 1]](%0, %conv.weight), scope: Model/Conv2d[conv]
  %3 : Float(1, 32, 98) = onnx::ReduceMean[axes=[2], keepdims=0](%2), scope: Model
  return (%3)

+ pytho verify_onnx_model.py
Traceback (most recent call last):
  File "/home/ubuntu/src/bin/testScripts/onnx_pytorch/assets/verify_onnx_model.py", line 13, in <module>
    assert inputs == ['0', '1']
AssertionError

```

## Expected behavior

Verification step should pass with no errors. Observing this behavior since updating PyTorch to 1.1

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):  1.1.0
 - OS (e.g., Linux): Ubuntu 16.04.6 LTS
 - How you installed PyTorch (`conda`, `pip`, source):  conda
 - Build command you used (if compiling from source):
 - Python version: 2.7 & 3.6
 - CUDA/cuDNN version: 10.0.130
 - GPU models and configuration:
GPU 0: Tesla K80
GPU 1: Tesla K80
GPU 2: Tesla K80
GPU 3: Tesla K80
GPU 4: Tesla K80
GPU 5: Tesla K80
GPU 6: Tesla K80
GPU 7: Tesla K80
GPU 8: Tesla K80
GPU 9: Tesla K80
GPU 10: Tesla K80
GPU 11: Tesla K80
GPU 12: Tesla K80
GPU 13: Tesla K80
GPU 14: Tesla K80
GPU 15: Tesla K80
 - Any other relevant information:

**Comment by kalyc** [comment]
Tested with ONNX 1.2.0 as well, latest Pytorch doesn't work with ONNX

**Comment by houseroad** [comment]
I didn't see any problem here. If you want to control the names of inputs in exported model, please check input_names parameter in export function: https://github.com/pytorch/pytorch/blob/master/torch/onnx/utils.py#L90

**Comment by darolt** [comment]
@houseroad I found the same issue yesterday.
I was using pytorch 1.0 and onnx 1.5 and everything was fine. After updating to 1.1, the conversion to .onnx results in this error:
RuntimeError: Failed to export an ONNX attribute, since it's not constant, please try to make things (e.g., kernel size) static if possible

**Comment by houseroad** [comment]
@darolt it's fixed in pytorch's master. If possible, could you try nightly build?

**Comment by kalyc** [comment]
Is this expected to be fixed in Pytorch 1.2?
